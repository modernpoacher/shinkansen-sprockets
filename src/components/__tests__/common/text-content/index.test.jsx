import React from 'react'
import renderer from 'react-test-renderer'

import Component from 'shinkansen-sprockets/components/common/text-content'

describe('shinkansen-sprockets/components/common/text-content', () => {
  describe('<Component />', () => {
    describe('With required props', () => {
      it('renders', () => {
        return expect(renderer.create(<Component textContent='MOCK TEXT CONTENT' />).toJSON())
          .toMatchSnapshot()
      })
    })
  })
})
