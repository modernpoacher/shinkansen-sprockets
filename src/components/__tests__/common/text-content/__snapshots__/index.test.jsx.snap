// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shinkansen-sprockets/components/common/text-content <Component /> With required props renders 1`] = `
<span
  className="text-content"
>
  MOCK TEXT CONTENT
</span>
`;
